.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+<!--
+   - SPDX-FileCopyrightText: 2019 Serokell <https://serokell.io>
+   -
+   - SPDX-License-Identifier: Unlicense
+   -->
+
+# Contribution Guidelines
+
+## Reporting Issues
+
+Please [open an issue](https://github.com/serokell/tezos-globacap/issues/new/choose)
+if you find a bug or have a feature request.
+Note: you need to login (e. g. using your GitHub account) first.
```

Comment:
Looks like GitHub does not provide a way to login using an account from another services, only GitHub account can be used, so at least `(e. g. using your GitHub account)` is useless. I'd remove this line fully to be honest.

Reply:
:yao: